docs: Physics srs overall description added (#123)
thoth-tech/documentation: docs/Splashkit/Modules/Physics/Background/Software_Requirements_Specification.md

### 2.1 User Needs

The user requirements for this module are that the module must be usable by game programmers that
use Splashkit. These requirements include:

> - Wrapper for the user's game to be implemented by the user

Reviewer: Is this true?
Surely only physics objects need to follow prescribed rules rather than the whole game/application.

Like how `apply_gravity(rectangle, time)` could add a downwards force to a rectangle would only require the existence of a rectangle and nothing else (i.e. no game wrapper used here)

Reviewer 2: We could make this "if applicable" in case there are multiple objects in the game (e.g. apply_gravity to an object in the game, but don't let it fall through another fixed object within the game)

- Each physics attribute can be implemented using a single method call
- Speed and weight of objects are able to be set by the user
- Use of physics methods on shapes, bitmaps and sprites.

### 2.2 Assumptions and Dependencies

This module has a few assumptions. These include:

> - Length measurements are all the same scale based on pixel ratio to metres
> - Default gravity is standardised to Earths value (9.8 m/s/s/).
> - Separate physics methods should be able to coexist without breaking.

Reviewer: ```suggestion
- Length measurements are all on the same scale of pixels per metre
- Default gravity is standardised to Earth's value (9.8 m/s/s)
- Separate physics methods should be able to coexist without breaking
```
